cmd/mungedocs/preformatted.go

```diff
@@ -45,7 +45,8 @@ func updatePreformatted(filePath string, mlines mungeLines) (mungeLines, error)
 // If the file ends on a preformatted line, there must have been an imbalance.
 func checkPreformatBalance(filePath string, mlines mungeLines) (mungeLines, error) {
 	if len(mlines) > 0 && mlines[len(mlines)-1].preformatted {
-		return nil, fmt.Errorf("file ends in preformatted block")
+		//		return mlines, fmt.Errorf("file ends in preformatted block: %s", mlines[len(mlines)-1].data)
+		return mlines, fmt.Errorf("file ends in preformatted block: %s", mlines[len(mlines)-1].data)
```

Comment:
Is "preformatted block" a well-known term? I think it'd better if we rephrase it as "cannot find the matching ```".